backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5688390_sys_gh15343_Fix_Calendrer_Remaining_Trl.sql

```diff
@@ -0,0 +1,97 @@
+-- Value: webui.calendar.resource.area.header
+-- 2023-05-17T20:26:34.212740Z
+INSERT INTO AD_Message (AD_Client_ID,AD_Message_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,MsgText,MsgType,Updated,UpdatedBy,Value) VALUES (0,545267,0,TO_TIMESTAMP('2023-05-17 21:26:33.896','YYYY-MM-DD HH24:MI:SS.US'),100,'de.metas.ui.web','Y','Ressource','I',TO_TIMESTAMP('2023-05-17 21:26:33.896','YYYY-MM-DD HH24:MI:SS.US'),100,'webui.calendar.resource.area.header')
+;
+
+-- 2023-05-17T20:26:34.217046800Z
+INSERT INTO AD_Message_Trl (AD_Language,AD_Message_ID, MsgText,MsgTip, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Message_ID, t.MsgText,t.MsgTip, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Message t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Message_ID=545267 AND NOT EXISTS (SELECT 1 FROM AD_Message_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Message_ID=t.AD_Message_ID)
+;
+
+-- Value: webui.calendar.resource.area.header
+-- 2023-05-17T20:26:58.921603500Z
+UPDATE AD_Message_Trl SET IsTranslated='Y', MsgText='Resource',Updated=TO_TIMESTAMP('2023-05-17 21:26:58.921','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Language='en_US' AND AD_Message_ID=545267
+;
+
+-- Value: webui.key.actual.data
```

Comment:
pls make sure all the sysconfig names are starting with `webui.calendar.` so as to have them organized.

Reply:
correction done at the end